spec/_global_parameters.yaml

```diff
@@ -10,21 +10,18 @@ components:
       description: Whether to pretty format the returned JSON response.
       schema:
         type: boolean
-      default: false
     human:
       name: human
       in: query
       description: Whether to return human readable values for statistics.
       schema:
         type: boolean
-      default: true
     error_trace:
       name: error_trace
       in: query
       description: Whether to include the stack trace of returned errors.
       schema:
         type: boolean
-      default: false
```

Comment:
We should move the default values into their respective schemas in this case.
We should also add `x-default` extension for parameters when the default for the param differs from the default of the schema OR when a shared schema itself doesn't have a default but certain param using said schema does.

Reply:
I did the first part, didn't realize that default should/can be nested under `schema` when I was removing this.

I don't think we have a case where the default differs, do you know of one?

Reply:
Haven't come across on where the schema itself has a default that is different from the param that uses it. But I've come across schemas that are used in different params of different operations but have different default values depending on the operation. But That was a long time ago, before we moved onto OpenAPI. But that's still a real possibility.